common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -109,7 +109,10 @@ eActionOp_CONTROL:
   ACTION_CONTROL_CHANNEL_SCAN_ABORT_NOTIFICATION: 146
   ACTION_CONTROL_CHANNEL_SCAN_FINISHED_NOTIFICATION: 147
 
-  ACTION_CONTROL_ENUM_END: 148
+  ACTION_CONTROL_MESSAGE_TO_RADIO_REQUEST: 148
+  ACTION_CONTROL_MESSAGE_TO_RADIO_RESPONSE: 149
```

Comment:
yamel and all related auto generated changes should be in separate "preparative" commit